File: swiftSchema/HouseUpdate.swift
import Foundation

struct HouseUpdate: Codable, Identifiable, Equatable {
	var id = UUID()

	var house_id: Int?
	var house_name: String?
	var unread_notifications: Int?
	var water_usage: Int?
	var electric_usage: Int?
	var water_temp: Int?
	var water_pressure: Int?
	var water_pressure_hourly_dip: Int?
	var water_usage_goal: Int?
	var electric_usage_goal: Int?
	var favorite: Int?
	var house_preference_id: Int?
	var last_seen: Int?

	private enum CodingKeys: String, CodingKey {
		case house_id, house_name, unread_notifications, water_usage, electric_usage, water_temp, water_pressure, water_pressure_hourly_dip, water_usage_goal, electric_usage_goal, favorite, house_preference_id, last_seen
	}
}

File: klaxonModels/HouseUpdate.kt
package io.homemetrics.homemetrics.schemas

import com.beust.klaxon.Klaxon

data class HouseUpdate(
	val house_id: Integer?,
	val house_name: String?,
	val unread_notifications: Integer?,
	val water_usage: Integer?,
	val electric_usage: Integer?,
	val water_temp: Integer?,
	val water_pressure: Integer?,
	val water_pressure_hourly_dip: Integer?,
	val water_usage_goal: Integer?,
	val electric_usage_goal: Integer?,
	val favorite: Integer?,
	val house_preference_id: Integer?,
	val last_seen: Integer?
)